tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -124,7 +125,7 @@ sub run_test {
         #reboot the guest
         record_info("VM reboot", "$guest");
         script_run "ssh root\@$guest 'reboot'";    #don't use assert_script_run, or may fail on xen guests
-        script_retry("nmap $guest -PN -p ssh | grep open", delay => 10, retry => 18, die => 1);
+        script_retry("nmap $guest -PN -p ssh | grep open", delay => 10, retry => 30, die => 1);
```

Comment:
May use `ensure_online()` here.

Reply:
thanks, found wait_guest_online() is more proper here. 'reboot' is expected to work, or the test is supposed to fail, so guest should not try to be virsh started again.